src/logic/operate.js

```diff
@@ -16,7 +16,7 @@ const operate = (numberOne, numberTwo, operation) => {
     case '%':
       return operandTwo.div('100').toString();
     case '÷':
-      if (operandTwo === 0) return 0;
+      if (operandTwo === 0) return "Error: You can't divide by 0";
```

Comment:
Division by zero is not handled properly yet. Let's fix this. 
The reason is that when a division by zero happens, what you did here is return `"Error: You can't divide by 0"` to `total`. This way, when a subsequent operation is performed on `total` this throws an error and crash the app.

Steps to reproduce:
- Divide a number by zero (e.g. 4 / 0)
- Click on any operator, e.g. +
The app crashes with `Error: [big.js] Division by zero`